File: Beginner/Java/1009/Main.java
import java.io.IOException;
import java.util.Scanner;

public class Main {

    public static void main(String[] args) throws IOException {
    	Scanner scanner = new Scanner(System.in);
        String name = scanner.nextLine();
        float fixSalary = scanner.nextFloat();
        float sales = scanner.nextFloat();
        
        double extra = sales * 0.15;
        
        
        System.out.printf("TOTAL = R$ %.2f%n", (fixSalary + extra));
    }

}

File: Beginner/C/1009/main.c
#include <stdio.h>
 
int main() {
    char name[50];
    double fixSalary, sales;
    scanf("%s", name);
    scanf("%lf", &fixSalary);
    scanf("%lf", &sales);

    double extra = sales * 0.15;

    printf("TOTAL = R$ %.2lf\n", (fixSalary + extra));
    return 0;
}
